core/primitives/src/time.rs

```diff
@@ -287,12 +287,12 @@ mod tests {
     }
 }
 
-mod time {
+mod time_struct {
```

Comment:
tbh, not sure this module pulls its weight at all. This is a realatively small (<1k lines) file, it feels like it doesnt' really need extra organization.